src/IP.php

```diff
@@ -62,12 +67,127 @@ abstract class IP
     /**
      * @var bool
      */
-    protected $is_private;
+    private $is_private;
+
+    /**
+     * @var array
+     */
+    protected static $privateRanges;
```

Comment:
As you can see, all variables in this lib uses snake_case formatting - please rename this one to match.

Reply:
OK.